src/sockperf.cpp

```diff
@@ -1714,7 +1714,13 @@ static int resolve_sockaddr(const char *host, const char *port, int sock_type,
         bool is_server_mode, sockaddr *addr, socklen_t &addr_len)
 {
     struct addrinfo hints;
+    char os_directory_path;
+#ifdef WIN32
+    os_directory_path = '\\';
+#else
+    os_directory_path = '/';
 
+#endif
```

Comment:
it is not useful because Windows does not support Unix Domain Socket. It can be removed.

Reply:
But it support AF_UNIX since 2018, so in the if statement ` if (path.size() > 0 && path[0] == os_directory_path)`
it will accept only for linux style, therefor we need to add definition for windows.

Reply:
I missed that, thanks. I see that windows does not support SOCK_DGRAM/AF_UNIX. Please check and kindly protect of this.

Reply:
Can we close this one?